### ISSUE TITLE
Declaring a variable as a function, within a class, causes TS Lint to make conflicting lints

### ISSUE THREAD

**Issue by dextermb** [palantir/tslint#3690]
### Bug Report

TS Lint seems to be unable to decide if a comma should be on the end of a variable defined as a function or not within a class. (Personally I wouldn't put a comma).

Is there an option for this that I am missing? Is there a better way of doing this? Or could a TS Lint option be added?

- __TSLint version__: `~5.7.0`
- __TypeScript version__: `2.6.2`
- __Running TSLint via__: `NodeJS API`  (Webstorm)

#### TypeScript code being linted

```ts
class Demo {
    private fn =  _ => {}; // My current tslint complains if there is a comma and if there isnt

    constructor(fn) {
        this.fn = fn;
    }
}
```

with `tslint.json` configuration:

```json
{
  "rulesDirectory": [
    "node_modules/codelyzer"
  ],
  "rules": {
    "arrow-return-shorthand": true,
    "callable-types": true,
    "class-name": true,
    "comment-format": [
      true,
      "check-space"
    ],
    "curly": true,
    "eofline": true,
    "forin": true,
    "import-blacklist": [
      true,
      "rxjs",
      "rxjs/Rx"
    ],
    "import-spacing": false,
    "indent": [
      true,
      "spaces"
    ],
    "interface-over-type-literal": true,
    "label-position": true,
    "max-line-length": [
      true,
      140
    ],
    "member-access": false,
    "member-ordering": [
      true,
      {
        "order": [
          "static-field",
          "instance-field",
          "static-method",
          "instance-method"
        ]
      }
    ],
    "no-arg": true,
    "no-bitwise": true,
    "no-console": [
      true,
      "debug",
      "info",
      "time",
      "timeEnd",
      "trace"
    ],
    "no-construct": true,
    "no-debugger": true,
    "no-duplicate-super": true,
    "no-empty": false,
    "no-empty-interface": true,
    "no-eval": true,
    "no-inferrable-types": [
      true,
      "ignore-params"
    ],
    "no-misused-new": true,
    "no-non-null-assertion": true,
    "no-shadowed-variable": false,
    "no-string-literal": false,
    "no-string-throw": true,
    "no-switch-case-fall-through": true,
    "no-trailing-whitespace": false,
    "no-unnecessary-initializer": true,
    "no-unused-expression": true,
    "no-use-before-declare": true,
    "no-var-keyword": true,
    "object-literal-sort-keys": false,
    "one-line": [
      true,
      "check-open-brace",
      "check-catch",
      "check-else",
      "check-whitespace"
    ],
    "prefer-const": true,
    "quotemark": [
      true,
      "single"
    ],
    "radix": true,
    "semicolon": [
      true,
      "always"
    ],
    "triple-equals": [
      true,
      "allow-null-check"
    ],
    "typedef-whitespace": [
      true,
      {
        "call-signature": "nospace",
        "index-signature": "nospace",
        "parameter": "nospace",
        "property-declaration": "nospace",
        "variable-declaration": "nospace"
      }
    ],
    "typeof-compare": true,
    "unified-signatures": true,
    "variable-name": false,
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type"
    ],
    "directive-selector": [
      true,
      "attribute",
      null,
      "kebab-case"
    ],
    "component-selector": [
      true,
      "element",
      null,
      "kebab-case"
    ],
    "use-input-property-decorator": true,
    "use-output-property-decorator": true,
    "use-host-property-decorator": true,
    "no-input-rename": false,
    "no-output-rename": false,
    "use-life-cycle-interface": true,
    "use-pipe-transform-interface": true,
    "component-class-suffix": true,
    "directive-class-suffix": true,
    "invoke-injectable": true
  }
}
```

#### Actual behavior

TS Lint complains if there is (or isn't) a comma after the variable `fn`.

#### Expected behavior

I'd like to be able to pick if there should be (or shouldn't) be a comma.

**Comment by dextermb** [comment]
The following fix resolves my issue:

```
    "semicolon": [
      true,
      "always",
      "strict-bound-class-methods"
    ],
```
